### PR TITLE
feature/pfqueue-golang

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Move pfqueue to go

# Issue
Fixes #ISSUE_NUMBER

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add OpenAPI specification
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Move pfqueue to go

### CODE REVIEW

File: go/pfconfigdriver/structs.go [hunk 1]
```diff
@@ -747,6 +747,51 @@ type PfConfRadiusConfiguration struct {
...
+	PfconfigMethod string                `val:"hash_element"`
+	PfconfigNS     string                `val:"config::Pfqueue"`
+	Pfqueue        Pfqueue               `json:"pfqueue"`
+	Queues         []Queue               `json:"queues"`
+	Consumer       Consumer              `json:"consumer"`
+	QueueConfig    map[string]QueueEntry `json:"queue_config"`
+	Producer       Producer              `json:"producer"`
+}
+
+type Pfqueue struct {
```
Comment (extrafu) [inline go/pfconfigdriver/structs.go]: Pfqueue -> PfQueue for consistency?

File: conf/systemd/packetfence-pfqueue-bankend.service [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright (C) Inverse inc.
+[Unit]
+Description=PacketFence pfqueue Service
```
Comment (fdurand) [inline conf/systemd/packetfence-pfqueue-bankend.service]: PacketFence pfqueue backend

File: conf/systemd/packetfence-pfqueue-bankend.service [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright (C) Inverse inc.
```
Comment (fdurand) [inline conf/systemd/packetfence-pfqueue-bankend.service]: not sure about the file name bankend ?!

File: conf/systemd/packetfence-pfqueue-go.service [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) Inverse inc.
+[Unit]
+Description=PacketFence pfqueue Service
```
Comment (fdurand) [inline conf/systemd/packetfence-pfqueue-go.service:3]: pfqueue go

**Review by fdurand — CHANGES_REQUESTED** [review]
It miss the packaging code and also the modification in pf::cmd::pf::service

**Pull request merged**